src/webgpu/api/operation/rendering/blending.spec.ts

```diff
@@ -279,7 +281,96 @@ g.test('formats')
     `Test blending results works for all formats that support it, and that blending is not applied
   for formats that do not. Blending should be done in linear space for srgb formats.`
   )
-  .unimplemented();
+  .params(u =>
+    u //
+      .combine('format', [
+        'r16float',
```

Comment:
I don't know how many formats this test should handle. Any suggestion?

Reply:
Thank you for finding out what the problem was in the test! But, there is an argument of type error when using 'kAllTextureFormats' or something simillar like below,

src/webgpu/api/operation/rendering/blending.spec.ts:362:55 - error TS2345: Argument of type '"r8unorm" | "r8snorm" | "r8uint" | "r8sint" | "r16uint" | "r16sint" | "r16float" | "rg8unorm" | "rg8snorm" | "rg8uint" | "rg8sint" | "r32uint" | "r32sint" | "r32float" | "rg16uint" | ... 26 more ... | "depth32float-stencil8"' is not assignable to parameter of type '"r8unorm" | "r8snorm" | "r8uint" | "r8sint" | "r16uint" | "r16sint" | "r16float" | "rg8unorm" | "rg8snorm" | "rg8uint" | "rg8sint" | "r32uint" | "r32sint" | "r32float" | "rg16uint" | ... 23 more ... | "depth32float"'.
  Type '"depth24plus"' is not assignable to type '"r8unorm" | "r8snorm" | "r8uint" | "r8sint" | "r16uint" | "r16sint" | "r16float" | "rg8unorm" | "rg8snorm" | "rg8uint" | "rg8sint" | "r32uint" | "r32sint" | "r32float" | "rg16uint" | ... 23 more ... | "depth32float"'.

362     const expTexelView = TexelView.fromTexelsAsColors(format, coords => expColor);
                                                          ~~~~~~

For now, it's likely `kEncodableTextureFormats` is only acceptable for the blending. wdyt?

Reply:
Thanks for catching that, that sounds like a correct fix to me.